kamon-statsd/src/main/resources/reference.conf

```diff
@@ -5,18 +5,10 @@
 kamon {
   statsd {
 
-    # Hostname and port in which your StatsD is running. Remember that StatsD packets are sent using UDP and
-    # setting unreachable hosts and/or not open ports wont be warned by the Kamon, your data wont go anywhere.
-    hostname = "127.0.0.1"
-    port = 8125
```

Comment:
do you guys want them like
```
statsd {
  hostname = "localhost"
  port = 8125
  batch-metric-sender {
    hostname = ${kamon.statsd.hostname}
    port = ${kamon.statsd.port}
    max-packet-size = 1024 bytes
  }
}
```
or just keep them on top level only? what about  `max-packet-size`